pkg/reconciler/autoscaling/kpa/resources/decider.go

```diff
@@ -54,14 +54,18 @@ func MakeDecider(ctx context.Context, pa *v1alpha1.PodAutoscaler, config *autosc
 	target, total := resources.ResolveConcurrency(pa, config)
 	panicThreshold := target * panicThresholdPercentage / 100.0
 
+	tbc := config.TargetBurstCapacity
+	if x, ok := pa.TargetBC(); ok {
+		tbc = x
+	}
```

Comment:
Can we align the pattern to the panicThresholdPercentage case? Just makes it easier to quickly parse that these two blocks are about defaulting logic.

Reply:
Done.